### DISCUSSION TITLE
how do i export my vercel analytics so i can share the data?

### DISCUSSION THREAD

**Discussion by rcolepeterson** [vercel/community#6488] [Help] [answered]
### Summary

I want to share my analytics data with my boss. I don't see a button on the Web Analytics page.

### Steps to Reproduce

I want to share my analytics data with my boss. I don't see a button on the Web Analytics page.

**Comment by amyegan** [comment]
Hey @rcolepeterson. Vercel Analytics data can't be exported at the moment. For now, it can only be viewed via the Vercel Dashboard.

Our Analytics team plans to include export features and API endpoints for Analytics data, however we don't currently have an ETA for when those will be available.

**Reply by amyegan** [reply, marked as answer]
You can export Analytics data as a CSV now 🎉 

https://vercel.com/changelog/csv-export-in-web-analytics

**Comment by rcolepeterson** [comment]
Ok. Seems crazy to me. Why would I use the service if I can't share
the data?

Am I missing something?

Cole

**Reply by versecafe** [reply]
@rcolepeterson there is actually a few hidden endpoints you can scrape the data out of if you need it before the release of official support

Example URL
*https://vercel.com/api/web-analytics/timeseries?environment=production&filter=%7B%7D&from=2024-04-07T00%3A00%3A00.000Z&projectId=psychometrica&teamId=REDACTED&to=2024-04-14T00%3A00%3A00.000Z&tz=America%2FNew_York*

Format:
`https://vercel.com/api/web-analytics/timeseries?environment=[env]&filter=[filter]&from=[date string ISO]&projectId=[project name]&teamId=[team ID]&to=[date string ISO]&tz=[time zone]`

the output data would look something like this

<img width="1331" alt="image" src="https://github.com/vercel/community/assets/147033096/24a9f266-785a-403c-b71d-c68c4556bf7d">

You can change out the parameters to include data as needed for this and apply any desired filters, do note this is not a public endpoint and has no stability guarantees but does allow you to export out your data.

**Comment by rcolepeterson** [comment]
Awesome, thanks for the info!

[a comment by an automated account is not shown]
